tanner/dbutils.py

```diff
@@ -0,0 +1,183 @@
+
+    @staticmethod
+    async def insert_queries(session, pg_client):
+        """Insert analyzed sessions into postgres
+
+        Arguments:
+            session {dict} -- dictionary having all the sessions details
+            pg_client {aiopg.pool.Pool}
+        """
+
+        def time_convertor(time):
+            """Convert the epoch time to the postgres
+            timestamp format
+
+            Arguments:
+                time {str} -- time in epoch format
+            """
+            return datetime.fromtimestamp(time).strftime('%Y-%m-%d %H:%M:%S')
+
+        Cookies = "INSERT INTO cookies(session_id, key, value) VALUES('{uuid}', '{key}', '{value}');"
+        Sessions = (
+            "INSERT INTO sessions (id, sensor_id, ip, port, country,"
+            "country_code, city, zip_code, user_agent, start_time,"
+            "end_time, rps, atbr, accepted_paths, errors, hidden_links, referer) "
+            "VALUES ('{uuid}','{sensor}','{ip}',{port},'{country}',"
+            "'{ccode}','{city}',{zcode},'{ua}','{st}','{et}',{rps},"
+            "{atbr},{apaths},{err},{hlinks},'{referer}');"
+        )
+        Paths = (
+            "INSERT INTO paths (session_id, path, created_at, response_status, attack_type) "
+            "VALUES ('{id}','{path}','{time}',{res},{atype});"
+        )
+        Owners = (
+            "INSERT INTO owners (session_id, owner_type, probability) "
+            "VALUES ('{id}', '{key}', '{val}');"
+        )
+
+        start_time = time_convertor(session["start_time"])
+        end_time = time_convertor(session["end_time"])
+
+        logger = logging.getLogger(__name__)
+
+        try:
+            sessions_query = Sessions.format(
+                uuid=session["sess_uuid"],
+                sensor=session["snare_uuid"],
+                ip=session["peer_ip"],
+                port=session["peer_port"],
+                country=session["location"]["country"],
+                ccode=session["location"]["country_code"],
+                city=session["location"]["city"],
+                zcode=session["location"]["zip_code"],
+                ua=session["user_agent"],
+                st=start_time,
+                et=end_time,
+                rps=session["requests_in_second"],
+                atbr=session["approx_time_between_requests"],
+                apaths=session["accepted_paths"],
+                err=session["errors"],
+                hlinks=session["hidden_links"],
+                referer=session["referer"]
+            )
+
+            async with pg_client.acquire() as conn:
+                async with conn.cursor() as cur:
+                    await cur.execute(sessions_query)
+                    print("Inserted sessions")
+                    for k, v in session["cookies"].items():
+                        await cur.execute(
+                            Cookies.format(uuid=session["sess_uuid"], key=k, value=v)
+                        )
+                    print("Inserted Cookies")
```

Comment:
maybe we can log instead of print?

Reply:
oh yes I was testing with these and was going to ask, should we log after every query? or should we just log once  like `Inserted sessions`

because if we log for every query then logs would contain lot of repetitive things because it will be repeated for all the sessions

Reply:
@mzfr I think we can remove this and log only exceptions :)

Reply:
I mean if we log only once then we can log with the session UUID so something like: `Inserted sessions with Session UUID <sess_uuid>`

it won't look much repetitive but if we log after every query then a lot of repetition would be there.

But also logging after every query would help us in debugging if something goes wrong :)

so I just wanted to ask your opinion on this.